app/scripts/modules/google/src/serverGroup/details/serverGroupDetails.html

```diff
@@ -106,6 +106,10 @@ <h4 class="text-center" ng-if="ctrl.serverGroup.isDisabled">[SERVER GROUP IS DIS
             <li ng-repeat="zone in ctrl.serverGroup.zones">{{zone}}</li>
           </ul>
         </dd>
+        <dt ng-if="ctrl.serverGroup.regional && ctrl.isAlphaListed()">Target Shape</dt>
+        <dd ng-if="ctrl.serverGroup.regional && ctrl.isAlphaListed()">
+          {{ctrl.serverGroup.distributionPolicy.targetShape}}
```

Comment:
If no `targetShape` is set, should we have some default text here like "No target shape set," or will it always default to `EVEN`?